rfcs/rfc0018.md

```diff
@@ -0,0 +1,105 @@
+# A feature to automatically "yield true" at the end of a file
+
+## Preamble
+
+    Author:  Curtis "Ovid" Poe <curtis.poe@gmail.com>
+    Sponsor:
+    ID:      0018
+    Status:  Draft
```

Comment:
I suggest to follow [the proposed self-assigned ID scheme](https://github.com/Perl/RFCs/blob/main/docs/future.md#rfc-ids-and-how-to-self-assign) as long as this remains a draft: `OVID-0018`

Reply:
Thank you. I hadn't seen that doc.

How does that translate to the filename? Does it matter?

Reply:
Good question. In this particular case, I expect the RFC will leave “Draft“ status and gain an official ID quickly, so the draft file name may not matter at all.

In general, however, continuing `rfc` with no delimiter seems less than ideal, particularly when the author chooses to use a (lowercase) GitHub ID. I think inserting a hyphen would make sense (`rfc-OVID-0018.md` / `rfc-ovid-0018.md`).